### PR TITLE
Add basic documentation on projects

### LINKED ISSUE #1557: Documentation on Projects missing
The documentation lacks any documentation on Projects which had been introduced with Nextcloud 16.

How is the feature to be used? 
What are the limitations?
Are there any APIs available?

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes #1557 

The last section already takes https://github.com/nextcloud/server/pull/16837 into account.

image

### CODE REVIEW

File: user_manual/files/projects.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+----------------------------------------------
+
+If another item should be added to an already existing project this can be done by searching for the project name in the *Add to a project* picker.
+
+.. figure:: ../images/projects4.png
+
+Visibility of projects
+----------------------
+
+Projects do not influence the access to the different items. Users will only see projects of other users if they have access to all containing items.
```
Comment (nickvergessen) [inline user_manual/files/projects.rst]:
```suggestion
Projects do not influence the access and visibility of the different items. Users will only see projects of other users if they have access to all contained items.
```

File: user_manual/files/projects.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+=======================
```
Comment (nickvergessen) [inline user_manual/files/projects.rst]:
```suggestion
========
```

File: user_manual/files/projects.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Projects
+=======================
```
Comment (nickvergessen) [inline user_manual/files/projects.rst]:
```suggestion
========
```

File: user_manual/files/projects.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Users can add files, chats and other items to projects. The various apps will present these items in a list, allowing users to immediately jump to them. Projects are Nextcloud wide, that is, when a user shares files that are part of a project, the other user can see the project with the shared files. A click on any of the items in a project leads right to it, be it a chat, a file or a task.
```
Comment (wiswedel) [inline user_manual/files/projects.rst]:
```suggestion
Users can associate files, chats and other items with each other in projects. The various apps will present these items in a list, allowing users to immediately jump to them. Projects are Nextcloud wide. When a user shares a file that is part of a project, the share recipient can see that project, too. A click on any of the items in a project leads right to it, be it a chat, a file or a task.
```

File: user_manual/files/projects.rst [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Create a new project
+-------------------------------
+
+A new project can be created by linking two items together. Start off by opening a file or folders sharing sidebar. 
+
+.. figure:: ../images/projects1.png
+
+After clicking the *Add to a project* section and selecting the type of item you want to link with the current file/folder a selector will open that allows you selecting a Talk conversation for example.
```
Comment (wiswedel) [inline user_manual/files/projects.rst]:
```suggestion
Click *Add to a project* select the type of item you want to link with the current file/folder. A selector will open that allows you selecting a Talk conversation for example.
```
shorter sentences

File: user_manual/files/projects.rst (hunk 4 above)
Comment (wiswedel) [reply user_manual/files/projects.rst]: resolved by https://github.com/nextcloud/documentation/commit/3c026c8d06353d0915e9dc048e655263d72476f0

**Pull request merged**